src/index.js

```diff
@@ -1,17 +1,16 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] `ESLint` is a tool for identifying and reporting on patterns found in ECMAScript/JavaScript code, with the goal of making code more consistent and avoiding bugs. Instead of disabling `eslint`, it will be nice to fix the issue.